### DISCUSSION TITLE
v2.4.0 craps out on OSX 12.1, v2.3.1 works?

### DISCUSSION THREAD

**Discussion by mfabache** [qutebrowser/qutebrowser#6929] [General] [answered]
I was using 2.3.1 for a while.  No issues.

I did an upgrade (just now 1/1/2022) from OSX 12.01 to 12.1
I also did a brew upgrade (which upgraded qutebrowser from 2.3.1 to 2.4.0)

2.4.0. would not load and only get a modal? window forcing me to move the app to the trash (tried this several times).
I reinstalled 2.3.1 and all is well.

Anyone else experiencing this issue or it could be my vanilla setup with brew.

Michael

**Comment by The-Compiler** [comment, marked as answer]
See #6771

**Reply by mfabache** [reply]
That solution totally worked and I am back in business with 2.4.0

Thank you Florian!
Happy New Year!

Best,
Michael
